tests/integration/linux_bridge_test.py

```diff
@@ -116,3 +117,21 @@ def test_create_and_remove_linux_bridge_with_two_ports(eth1_up, eth2_up):
         assertlib.assert_state(desired_state)
 
     assertlib.assert_absent(bridge_name)
+
+
+def test_linux_bridge_mac_with_one_iface_uses_default_mac_addr(eth1_up):
```

Comment:
The actual test is hiding deep in the code here.
Could you please consider re-basing this over https://github.com/nmstate/nmstate/pull/370 ?